src/tasks/sep/047-soneium-add-game-type/config.toml

```diff
@@ -0,0 +1,41 @@
+templateName = "AddGameTypeTemplate"
+
+l2chains = [    
+    {name = "Soneium Testnet Minato", chainId = 1946},
+]
+
+[[configs]]
+# The below addresses can be found in the Superchain Registry https://github.com/ethereum-optimism/superchain-registry/blob/851f7bb60edc53373fc961d237c7175aa130fba3/superchain/extra/addresses/addresses.json#L402
+chainId = 1946
+saltMixer = "this is a salt mixer"
+systemConfig = "0x4Ca9608Fef202216bc21D543798ec854539bAAd3"
+proxyAdmin = "0xff9d236641962Cebf9DBFb54E7b8e91F99f10Db0"
+delayedWETH = "0xB39c1730DFF54f25F9e45667c119e0a8FeE73156"
+disputeGameType = 0
+disputeAbsolutePrestate = "0x03caa1871bb9fe7f9b11217c245c16e4ded33367df5b3ccb2c6d0a847a217d1b" # Absolute Prestate 1.8.0 https://github.com/ethereum-optimism/superchain-registry/blob/9205709c8823c65cdb3d186b9afa7d8effb943a8/validation/standard/standard-prestates.toml#L12
```

Comment:
This prestate is correct, it has been reviewed for soneium-minato-sepolia [here](https://www.notion.so/oplabs/U17-2a1f153ee162804f94d0c10d0a1e1258?v=1bdf153ee1628012962d000c0011a34a&pvs=25)